### PR TITLE
Fix rounded borders on author block

### PR DESCRIPTION
**Pull request by ooflorent**
**Before:**

before

**After:**

after

### CODE REVIEW

File: web_modules/_legacy-css/blocks/author.css [hunk 1]
```diff
@@ -90,6 +88,7 @@
     }
 
     .putainde-Author-social a {
+      background: #f9f9f9;
```
Comment (ooflorent) [inline web_modules/_legacy-css/blocks/author.css]: Previous one was too light.

File: web_modules/_legacy-css/blocks/author.css [hunk 2]
```diff
@@ -99,8 +98,19 @@
       .putainde-Author-social a .putainde-Icon { font-size: 1.2em; }
       @media (--r-maxS) { .putainde-Author-social a { font-size: 1em } }
 
-      .putainde-Author-social a:hover,
-      .putainde-Author-social a:focus { background: #eee }
+      .putainde-Author-social a:active,
```
Comment (ooflorent) [inline web_modules/_legacy-css/blocks/author.css]: Added `:active`

**Comment by MoOx** [comment]
lgtm

**Comment by Nyalab** [comment]
:shipit:

**Comment by magsout** [comment]
LGTM :shipit:

**Pull request merged**